src/test/java/uk/gov/ons/ctp/integration/ratelimiterclient/RateLimiterClientTest.java

```diff
@@ -65,6 +66,18 @@ public void initMocks() {
     MockitoAnnotations.initMocks(this);
```

Comment:
FYI remove this and replace with class level annotation
```
@RunWith(MockitoJUnitRunner.class)
```
adds some useful functionality for eliminating dead mocks in the code. Well worth doing.

Reply:
Done